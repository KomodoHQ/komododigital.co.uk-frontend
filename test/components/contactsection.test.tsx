import React from 'react';
import ContactSection from '../../src/components/contactsection';
import renderer from 'react-test-renderer';

test('ContactSection element is rendered', () => {
  const component = renderer.create(<ContactSection />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
