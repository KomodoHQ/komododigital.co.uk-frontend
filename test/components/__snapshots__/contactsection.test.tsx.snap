// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSection element is rendered 1`] = `
<div
  className="komodoGridWrapper contactsection-wrapper"
>
  <div
    className="contactsection"
  />
</div>
`;
